Dockerfile

```diff
@@ -21,7 +21,7 @@ COPY . .
 # Build the final image.
 ##########################
 
-FROM alpine:latest
+FROM --platform=$TARGETPLATFORM alpine:latest
```

Comment:
Change `--platform=$TARGETPLATFORM` to `--platform=linux/$ARCH` and I think this will work.